CHANGELOG.md

```diff
@@ -94,6 +94,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1974**
   - Fixed @cumulus/api webpack config for missing underscore object due to underscore update
 
+- **CUMULUS-1987**
```

Comment:
These CHANGELOG notes need to be moved out from under release 1.22.1 to be under Unreleased

Reply:
Thank you! I think that was a merge gone bad. This has been fixed.